### ISSUE TITLE
in docs/forms/select, the default selected option tag should have an aria-hidden="true"

### ISSUE THREAD

**Issue by Aniort** [twbs/bootstrap#37614]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

hello,
in docs/forms/select, the default selected option tag should have an aria-hidden="true" to avoid a vocalization of the content of the option when an AT is entering the select tag. This content is not really needed and the label of the select is enough to understand his purpose.
And this has to be made in every select tag of the docs...

### Reduced test cases

https://getbootstrap.com/docs/5.0/forms/select/

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What version of Bootstrap are you using?

v5.0 and down

**Comment by Aniort** [comment]
hello,
you should modify each default option tag with the text content of "Open this select menu" by putting :
<select class="form-select" aria-label="Disabled select example" disabled>
  <option selected **_aria-hidden="true"_**>Open this select menu</option>
  <option value="1">One</option>
  <option value="2">Two</option>
  <option value="3">Three</option>
</select>
To be aware of such a modification, you can see it in the accessibility panel in the devtool of Firefox. When adding the ariâ-hidden="true", the value of the select tag must be null
![image](https://user-images.githubusercontent.com/13222389/206670681-3f930d24-2e03-411a-a680-c691a757dc5e.png)
and in Chrome, you can see the null value in the accessibility subpanel in devtool
![image](https://user-images.githubusercontent.com/13222389/206676310-e0b4f030-c3e9-45d9-8216-fa00853d1eaa.png)
if you need more infos, just tell me !

**Comment by UnfixedSun** [comment]
I have raised the PR #37619 and please find attached the screenshot of the validation that I have done in chrome.
![image](https://user-images.githubusercontent.com/113256893/206748826-365c2a6e-b0b4-4792-aceb-63b39d2c1725.png)
